### PR TITLE
test 016

### PR DESCRIPTION
**Pull request by manojnarsikar**
hi team,
created test script for test 016, please verify.

### CODE REVIEW

File: src/test/java/com/Test/Test_TC_016_m.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.Test;
+
+import org.testng.annotations.Test;
+
+public class Test_TC_016_m {
+	@Test
+	public void VerifyTest() {
+		System.out.println("Test_TC_016_m");
```
Comment (GovindPatil75) [inline src/test/java/com/Test/Test_TC_016_m.java:8]: please verify result of an operation

**Review by GovindPatil75 — COMMENTED** [review]
comment added

File: src/test/java/com/Test/Test_TC_016_m.java (hunk 1 above)
Comment (manojnarsikar) [reply src/test/java/com/Test/Test_TC_016_m.java:8]: we already added check

Comment (GovindPatil75) [reply src/test/java/com/Test/Test_TC_016_m.java:8]: looks good .approved

**Pull request merged**